lib/rex/post/meterpreter/packet_dispatcher.rb

```diff
@@ -130,8 +131,7 @@ def send_packet(packet, opts={})
       tlv_enc_key = opts[:tlv_enc_key]
     end
 
-    # Uncomment this line if you want to see outbound packets in the console.
-    # STDERR.puts("\n\e[1;31mSEND\e[0m: #{packet.inspect}\n")
+    log_packet(packet, :outbound)
```

Comment:
I prefer your name, but for consistency:

```suggestion
    log_packet(packet, :send)
```